src/Catty/ViewController/Upload/UploadViewController.swift

```diff
@@ -81,62 +84,74 @@ class UploadViewController: UIViewController {
         initDescriptionViewElements()
         initObservers()
         hideKeyboardWhenTapInViewController()
-
         self.uploadBarButton = UIBarButtonItem(title: kLocalizedUploadProject,
                                                style: .plain,
                                                target: self,
                                                action: #selector(UploadViewController.checkProjectAction))
         navigationItem.rightBarButtonItem = self.uploadBarButton
-
-        projectNameTextField.becomeFirstResponder()
     }
 
     // MARK: - Initialization
 
     required init?(coder aDecoder: NSCoder) {
+        self.seperationView = [UIView]()
+        self.labels = [UILabel]()
+        self.values = [UILabel]()
+        self.projectNameTextField = UITextField()
+        self.descriptionTextView = UITextView()
         super.init(coder: aDecoder)
         self.project = Project.init(loadingInfo: Util.lastUsedProjectLoadingInfo())!
     }
 
     func initProjectNameViewElements() {
-        projectNameLabel.textColor = UIColor.globalTint
-        projectNameLabel.text = kLocalizedName
-        projectNameLabel.font = UIFont.boldSystemFont(ofSize: uploadFontSize)
-
-        projectNameTextField.borderStyle = .roundedRect
-        projectNameTextField.layer.borderWidth = 1.0
-        projectNameTextField.layer.borderColor = UIColor.textViewBorderGray.cgColor
-        projectNameTextField.layer.cornerRadius = 3
+        addLineViewElement(withTopConstrain: 3 * verticalConstrainValue, fromElement: self.view)
+        createLabel(text: kLocalizedName, textColor: UIColor.globalTint, font: .boldSystemFont(ofSize: labelFontSize))
```

Comment:
Please directly return the created label from this function and replace the "if lets" below with the actual label.